test/unit/Reflection/ReflectionFunctionAbstractTest.php

```diff
@@ -133,6 +136,25 @@ public function testIsDeprecated() : void
         self::assertFalse($function->isDeprecated());
     }
 
+    public function testStubbedIsDeprecated(): void
+    {
+        BetterReflection::$phpVersion = 70400;
+        $reflector = new FunctionReflector(
+            new PhpInternalSourceLocator($this->astLocator, new PhpStormStubsSourceStubber($this->parser, BetterReflection::$phpVersion)),
+            $this->classReflector
+        );
+        $function = $reflector->reflect('create_function');
+        self::assertTrue($function->isDeprecated());
+
+        BetterReflection::$phpVersion = 70100;
```

Comment:
Instead of duplicated code like that, it'd be nicer to do this via a `@dataProvider`.